src/accordion/Accordion.tsx

```diff
@@ -0,0 +1,38 @@
+import "./_accordion.scss";
+
+import React from "react";
+import classNames from "classnames";
+
+import AccordionItem from "./item/AccordionItem";
+import {AccordionContextProvider} from "./AccordionProvider";
+
+export type AccordionProps = {
+  allowMultipleExpanded?: boolean;
+  allowZeroExpanded?: boolean;
+  defaultExpanded?: string[];
+  customClassName?: string;
+  children: React.ReactNode;
+};
+
+function Accordion(props: AccordionProps) {
+  const {
+    allowMultipleExpanded,
+    allowZeroExpanded,
+    defaultExpanded,
+    customClassName,
+    children
+  } = props;
```

Comment:
If we are using all of the props, we don't need destructing on another constant. 
```suggestion
function Accordion({
    allowMultipleExpanded,
    allowZeroExpanded,
    defaultExpanded,
    customClassName,
    children
  }: AccordionProps) {
```

If there are some props that we don't use directly, we can destruct props that we are using, and we use `...rest` for another props.